Implemented instructions and base app

jordanhudgens opened the pull request:
@AnnJohn Let me know how you'd like me to submit the solution branch, you can see it [here](https://github.com/jordanhudgens/rails-form_for-lab/tree/solution)

jmburges commented on README.md:
```diff
@@ -1,16 +1,41 @@
- - make sure to use form_for.
+## Instructions
+
+For this lab you will be implementing all of the functionality from scratch, including building out the models, views, and controllers for the `SchoolClass` and `Student` resources.
```
Give a high level overview of what they'll be building. Like what is the app.

jmburges commented on spec/factories/school_classes.rb:
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```
No Factories

jmburges commented:
back at you

jmburges commented:
disable strong params!

jordanhudgens commented:
@jmburges I updated the content, removed FG and disabled strong params, I also updated the [solution](https://github.com/jordanhudgens/rails-form_for-lab/tree/solution) branch and ensured all of the tests are still passing there.
